azkaban-web-server/src/main/java/azkaban/webapp/servlet/PageUtils.java

```diff
@@ -0,0 +1,25 @@
+package azkaban.webapp.servlet;
+
+import azkaban.server.session.Session;
+import azkaban.user.Permission;
+import azkaban.user.User;
+import azkaban.user.UserManager;
+import azkaban.user.UserUtils;
+
+
+public final class PageUtils {
+
+  private PageUtils() {
+
+  }
+
+  public static void hideUploadButtonWhenNeeded(final Page page, final Session session, final UserManager userManager,
+      final Boolean lockdownUploadProjects) {
+    final User user = session.getUser();
+
+    if (lockdownUploadProjects || !UserUtils.hasPermissionforAction(userManager, user,
```

Comment:
It looks like when the existing lockdown.upload.projects config is set, no one can upload projects even for AZ admin. 

However lockdown.create.projects config behaves differently, it allows AZ admins or users with the create project permission to create projects.

    if (this.lockdownCreateProjects &&
        !UserUtils.hasPermissionforAction(this.userManager, user, Type.CREATEPROJECTS)) {

It seems to me this model makes more sense. The uploadproject permission only applies when lockdown is set. 

If you choose this model, it may help to add a comment to this permission to explain this relationship.

Thoughts?